CONTRIBUTING.md

```diff
@@ -68,7 +68,7 @@ with your real name (first and last name)
 If you set your `user.name` and `user.email` git configs, you can sign your
 commit automatically with `git commit -s`. If you forgot this you can
 use `git commit -s --amend` to add this in retrospective for the last commit.
-If you need to sign-off multiple commits within a branch, you need to do an interactive
+If you need to sign off multiple commits within a branch, you need to do an interactive
```

Comment:
Please keep `sign-off` . It implies user to use `git commit --sign-off`

Reply:
ok

Reply:
I changed that @rohanKanojia